spring-boot-example/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is this autogenerated?

Reply:
Yes, autogenerated by the wrapper script. This file is required to execute `./mvnw`.